Tests/SwiftDocCTests/Rendering/TermListTests.swift

```diff
@@ -46,6 +47,112 @@ class TermListTests: XCTestCase {
         XCTAssertEqual(l.items.count, 4)
     }
     
+    func testLinksAndCodeVoiceAsTerms() throws {
+        let tempURL = try createTempFolder(content: [
+            Folder(name: "unit-test.docc", content: [
+                TextFile(name: "Article.md", utf8Content: """
+                # Article
+                
+                An article with a term definition list with links as terms.
+                
+                - term ``someFunction(_:)``: First definition
+                - term <doc:ModuleName>: Second definition
+                - term `someFunction(_:)`: Third definition
+                - term <doc://unit-test/documentation/ModuleName>: Fourth definition
+                - term <doc://com.external.testbundle/path/to/something>: Fifth definition
+                                                
+                """),
+            
+                JSONFile(name: "ModuleName.symbols.json", content: makeSymbolGraph(
+                    moduleName: "ModuleName",
+                    symbols: [
+                        .init(
+                            identifier: .init(precise: "some-symbol-id", interfaceLanguage: SourceLanguage.swift.id),
+                            names: .init(title: "someFunction(_:)", navigator: nil, subHeading: nil, prose: nil),
+                            pathComponents: ["someFunction(_:)"],
+                            docComment: nil,
+                            accessLevel: .public,
+                            kind: .init(parsedIdentifier: .func, displayName: "Kind Display Name"),
+                            mixins: [:]
+                        )
+                    ]
+                )),
+            ]),
+        ])
+            
+        let resolver = TestMultiResultExternalReferenceResolver()
+        resolver.entitiesToReturn["/path/to/something"] = .success(
+            .init(referencePath: "/path/to/something")
+        )
+        
+        let (_, bundle, context) = try loadBundle(from: tempURL, externalResolvers: ["com.external.testbundle": resolver])
+        
+        let reference = ResolvedTopicReference(bundleIdentifier: bundle.identifier, path: "/documentation/unit-test/Article", sourceLanguage: .swift)
+        let entity = try context.entity(with: reference)
+        
+        let converter = DocumentationNodeConverter(bundle: bundle, context: context)
+        let renderNode = try converter.convert(entity, at: nil)
+        
+        let overviewSection = try XCTUnwrap(renderNode.primaryContentSections.first as? ContentRenderSection)
+        
+        guard case RenderBlockContent.termList(let termList) = try XCTUnwrap(overviewSection.content.dropFirst(/* the "Overview" heading */).first) else {
+            XCTFail("Unexpected kind of rendered content. Expected term list. Got \(overviewSection.content.dropFirst().first ?? "<nil>")")
+            return
+        }
+        
+        XCTAssertEqual(termList.items.count, 5)
+        
+        do {
```

Comment:
Why write these `do { }` blocks? I don't see any `try` keywords.

Reply:
I use them for scope; partly so that each item can have a variable named `item` and partly so that it's clearer which assertions go together with which item